src/patternfly/components/Tabs/tabs.hbs

```diff
@@ -0,0 +1,20 @@
+<div class="pf-c-tabs{{#if tabs--modifier}} {{tabs--modifier}}{{/if}}">
```

Comment:
can you add a block for `{{tabs--attribute}}`?

Reply:
Added.